packages/flutter/lib/src/animation/curves.dart

```diff
@@ -60,6 +60,8 @@ class SawTooth extends Curve {
 
   @override
   double transform(double t) {
+    if (t == 1.0)
+      return 1.0;
```

Comment:
This shouldn't be necessary.  The consumers of the curve are the ones required to hit 1.0 exactly.